test/samples_test.dart

```diff
@@ -0,0 +1,48 @@
+library samples_test;
+
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:csslib/parser.dart';
+import 'package:csslib/src/messages.dart';
+import 'testing.dart';
+
+const testOptions = const PreprocessorOptions(
+    useColors: false,
+    checked: false,
+    warningsAsErrors: true,
+    inputFile: 'memory'
+);
+
+void testCSSFile(String name) {
+  var errors = <Message>[];
+  File bootstrap = new File('test/examples/${name}.css');
```

Comment:
The path here only works if the tests are run from the root directory of the project. Consider making this relative to this file's path?

Furthermore we could just read all the files from the _examples/_ directory and test each one rather than having to specify them here manually. That way adding new examples is even easier.